td-agent/yum/td-agent.spec.in

```diff
@@ -25,6 +25,14 @@
 %define __arch_install_post %{nil}
 %undefine __brp_mangle_shebangs
 
+%if %{use_systemd}
+%define control_service() (systemctl %1 %2)
```

Comment:
Should we use full path here?

Reply:
No, systemctl is placed at /usr/bin/, so no need to use full path here.